ds-collector/README.md

```diff
@@ -0,0 +1,65 @@
+ds-collector
+=============
+
+This script will connect to a target host generate an artifact bundle and transfer that bundle back to the execution host. From there they are pushed to a "dead drop" S3 bucket using HTTPS.
+
+The package contains the three files:
+
+* `ds-collector` the script that does the work.
+* `collector.conf` configuration settings, see help in the file.
+* `collector.hosts` optional list of hosts to get information from.
+
+
+Usage
+-----
+
+```
+usage: ds-collector [option1] [option2] [option...] [mode]
+
+options:
+-n=[HOST_IP | HOST_NAME | HOST_FILE_PATH]   IP address or hostname of Cassandra node to obtain
+                                              the list of hosts to run collector on. Can also
+                                              specify path to file with list of nodes or
+                                              ip addresses.
+-f=CONFIG_FILE_PATH     Path to a configuration file. Will override all defaults and options.
+-a=ARTIFACT_PATH        Path to single artifact to upload to s3.
+-p                      Ping the host prior to connecting to it. Can only be used with Test Mode.
+-d                      Run script on a single node only, disabling auto discovery when the -n
+                          option specifies a hostname or IP address. Can only be used with
+                          Test Mode and Execute Mode.
+-h                      Print this help and exit.
+
+modes:
+-T  Test Mode:      Complete a connection test.
+-X  Execute Mode:   Execute collector on a cluster using arguments passed above.
+-C  Client Mode:    Execute collector for only the node this script is run on.
+```
+
+Directions
+----------
+
+* Extract the three files to your central host (e.g. bastion or jumpbox).
+* Copy the generated encryption key into the same directory as the ds-collector script.
+* Make sure the correct permissions are set on the ds-collector so it is executable.
+    ```
+    chmod +x ds-collector
+    ```
+* update `collector.conf` variable "userName" to the name of a user who can log into the nodes.
```

Comment:
Here we switch to lower case letters starting the sentences, but that's just cosmetics.